app/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -58,10 +58,6 @@
     <string name="tap_on_mic">Toque no microfone para falar</string>
     <string name="know_more">Saiba mais</string>
 
-    <string name="settings_theme">Configurações do Tema</string>
-    <string name="theme">Selecionar Tema</string>
-    <string name="selection_theme"></string>
-
     <!--Error Messages-->
     <string name="error_crop_not_supported">Desculpe - seu dispositivo não suporta a ação de cortar!</string>
```

Comment:
@collinx Can you please remove this line as well from both translations. This one 
`<string name="error_crop_not_supported">Desculpe - seu dispositivo não suporta a ação de cortar!</string>`